addons/sale_management/tests/test_sale_order.py

```diff
@@ -431,3 +431,26 @@ def get_form_field_names(form):
             names_NL,
             "Lines shouldn't change once saved",
         )
+
+    def test_warning_quotation(self):
+        """
+        ensure "warning for the change of your quotation's company" isn't triggered
+        during the creation of a quotation when a quotation template is set as default
+        """
+        quotation_template = self.env['sale.order.template'].create({
+        'name': 'Test Template',
+        'sale_order_template_line_ids': [Command.create({
+            'product_id': self.env['product.product'].create({
+                'name': 'Test Product',
+                'list_price': 100.0
+                }).id,
+                'product_uom_id': 1
+            })]
+        })
```

Comment:
```suggestion
        quotation_template = self.empty_order_template
        quotation_template.sale_order_template_line_ids = [
            Command.create({'product_id': self.product.id})
        ]
```
should have said it in my first review:
* never use hardcoded record ids (`product_uom_id`), you never know whether the record was deleted or not, maybe recreated, ...
* the indentation was wrong
* you should use the existing data from the commons to speed up and simplify your test setups